website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -176,4 +176,64 @@ Inline alerts can be added to a section or component or inline with content.
 
 Compact alerts can be added to a section or component or inline with content.
 
-![Placement of compact Alert](/assets/components/alert/alert-placement-compact.png =600x*)
\ No newline at end of file
+![Placement of compact Alert](/assets/components/alert/alert-placement-compact.png =600x*)
+
+## Multiple alerts
+
+In general, displaying multiple alerts on the same page is discouraged as it can negatively impact the page hierarchy and unnecessarily shift emphasis away from the main content on the page.
+
+However, depending on the logic within the application and the type content being expressed to the user, it may be necessary to display more than one Alert on a page. Consider these scenarios and their accompanying guidelines if this is the case.
+
+### Alerts of the same color
+
+!!! Dont
+
+Don’t display more than one of the same `color` Alert on the page. This adds unnecessary visual weight to page and may result in the user being confused about which Alert is more important or which might require attention first.
+
+![Example of multiple Alerts of the same color](/assets/components/alert/multiple-color-alerts-dont.png)
+!!!
+
+!!! Do
+
+Instead, try to combine relevant and overlapping content into a single Alert.
+
+![Example of multiple Alerts combined into one](/assets/components/alert/multiple-color-alerts-do.png)
+
+This may result in the length of the content being longer, but consolidating messages into a single instance of the Alert is a more concise communication pattern.
+!!!
+
+### Ordering of Alerts
+
+Order multiple Alerts by their importance and impact on the user, with `critical` communicating Alerts that must be resolved immediately, and descending in perceived impact to `neutral`.
+
+![Ordering alerts](/assets/components/alert/ordering-alerts.png)
+
+### Alerts of the same type
+
+Try to avoid multiple Alerts of the same type (`page`, `inline`, etc) on the same page.
+
+!!! Do
+
+Instead, use Alerts hierarchically according to their intended prominence and importance to the user.
+
+![An example of multiple types of Alerts on the same page](/assets/components/alert/multiple-alert-types-do.png)
+!!!
+
+### Contextualizing Alerts
+
+`Inline` and `compact` Alerts can have more meaning if they are [placed](#placement) or contextualized within the element that is responsible for the Alert. This can help when it's necessary to have more than one Alert on the page and is relevant for pages that aggregate content like dashbaords.
+
+![Example of multiple Alerts contextualized to different parts of the UI](/assets/components/alert/combining-contextualized-alerts.png)
+
+### Alerts within a Reveal
+
+**As a last resort**, when multiple `inline` Alerts are necessary, display the most important Alert the user within the page, and obfuscate additional Alerts within a [Reveal](/components/reveal).
```

Comment:
I remember when we implemented the Alert in HDS, there were "stacked alerts" in Structure (imagine like stacked cards). Should we consider this as a (future) super-component to solve this pattern?

Reply:
Yeah maybe, though it seems like it's pretty easy to build just using our components. I guess the tricky part is probably more around the logic for displaying them in this way, thinking:

- if alerts.length =< 1 then render the alert directly in the page
- if alerts.length > 1 then render alert[0] in the page and everything else in a reveal

Could get a little tricky, but might be helpful to have this as a HDS+ pattern 😉

Reply:
[This](https://www.figma.com/file/B7yd8loYS7nA2TiQcJ8UtV/Structure-1.9-%F0%9F%9A%A8-FULLY-DEPRECATED-%F0%9F%9A%A8?type=design&node-id=12569-15900&mode=design&t=lSbTh6sVKZ6zZzgl-4) is the only thing I was able to find in Structure, which appears to be the same as what Jory is proposing. @didoo I imagine what you're suggesting is something like this:

<img width="1068" alt="collapsed-alerts" src="https://github.com/hashicorp/design-system/assets/8553306/99ea927a-7387-4491-9fdf-5d8d12001237">

I'm curious about this direction because it gives the user more context on the severity of the alerts below which feels more informative and may help the user avoid missing necessary notifications.

Reply:
@heatherlarsen unfortunately I can't get Storybook run on an old version of Structure to compare with that design, but I think it was something similar (can't remember what happened in terms of UI transition, when the user clicked on the "additional alerts").